test/error/ErrorTest.ts

```diff
@@ -0,0 +1,185 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import assert from "assert";
+import { MatterError } from "../../src/error/MatterError";
+import { StatusCode } from "../../src/matter/interaction/InteractionMessages";
+import { tryCatch, tryCatchAsync } from "../../src/error/TryCatchHandler";
+import { StatusResponseError } from "../../src/matter/interaction/InteractionErrors";
+
+class SubMatterError extends MatterError {}
+class SubSubMatterError extends SubMatterError {}
+class OtherMatterError extends MatterError {}
+
+describe("Errors", () => {
+
+    context("Test tryCatch method", () => {
+        it("tryCatch without error return value", () => {
+            const error = tryCatch(() => {
```

Comment:
There is some indentation issues here that makes this code block hard to read.
Current:
```
const error = tryCatch(() => {
    throw new SubMatterError("test");
},
SubMatterError, "caught");
```

Suggested:
```
const error = tryCatch(() => {
        throw new SubMatterError("test");
    },
    SubMatterError, "caught"
);
```

(same in the rest of the class)